microprofile/tests/tck/tck-metrics/pom.xml

```diff
@@ -131,6 +131,10 @@
                     <systemPropertyVariables>
                         <context.root/> <!-- Suppress the TCK's default of /optionalTCK -->
                         <metrics.rest-request.enabled>true</metrics.rest-request.enabled> <!-- off by default -->
+                        <!-- TODO - remove setting of gc-time-type for Helidon 5
```

Comment:
Can you please use a different way to mark things to be removed in 5?
For example `@Deprecated(forRemoval = true)` , so we can just search for this string.
TODO means that we missed something...

Reply:
OK. 

I guess to me, TODO means exactly that--we know something needs to be done in the future. Yes, sometimes people use it to flag omitted work that should already have been done, but that is not true of the TODOs here and I believe the text in each clearly indicates why they are there. 

But I'll go ahead and change the TODOs for removal in 5.x to commented annotations.